buildenv/jenkins/common/pipeline-functions

```diff
@@ -152,6 +152,9 @@ def workflow() {
                 case "_extended":
                     TEST_JOB_NAME = "Test-Extended-JDK${SDK_VERSION}-${SPEC}"
                     break
+                case "_experimental":
```

Comment:
I do not agree with adding a new top-level test make target.  This implies that all Valhalla test playlist.xml files have been modified also, and would also require some overhaul of test documentation at both OpenJ9 and Adopt.  It also opens the door to others tagging/categorizing tests as experimental, and I am reserving the right to control whether or not we introduce more top-level test make targets.